Source/engine/demomode.cpp

```diff
@@ -290,7 +296,7 @@ void NotifyGameLoopEnd()
 	}
 
 	if (IsRunning()) {
-		float secounds = (SDL_GetTicks() - StartTime) / 1000.0;
+		float secounds = (SDL_GetTicks() - StartTime) / 1000.0f;

```

Comment:
```suggestion
		float seconds = (SDL_GetTicks() - StartTime) / 1000.0f;
```
pls fix while we are at it :D